frontend/packages/kubevirt-plugin/integration-tests-cypress/support/virtualization.ts

```diff
@@ -0,0 +1,47 @@
+import { createYAMLButton, nameFilter, resourceTitle } from '../views/selector';
+import { create, templateYAML, vmYAML } from '../views/selector-wizard';
+
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      visitVirt(): Chainable<Element>;
+      visitVirtTemplate(): Chainable<Element>;
+      filterByName(name: string): Chainable<Element>;
+      createDefaultVM(): Chainable<Element>;
+      createDefaultTemplate(): Chainable<Element>;
+    }
+  }
+}
+
+// any command added below, must be added to global Cypress interface above
+
+Cypress.Commands.add('visitVirt', () => {
```

Comment:
I like this idea! My only concern is around naming. We're going to be adding the Virtualization Overview page soon and moving the VMs and Templates pages to their own dedicated pages instead of tabs, so this won't work anymore since there won't be an actual Virtualization page that defaults to the VMs list. What do you think about the following? 

- `visitVirtualizationOverview` (when it's added)
- `visitVMsList`
- `visitVMTemplatesList`

Reply:
`visitVMsList` and `visitVMTemplatesList` looks better, will change the name.